build/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## December 2020
+- [Changed] Updated watch directory folder in `Development.ps1` script to match official Sitecore 10 images. Mount directories in existing compose files should be changed to the updated location.
```

Comment:
Very minor, but this probably should be noted as "breaking". Anyone using <= 10 community images have all of their docker-compose setups done with a mapping to c:\src. The fact that it changed in 10 doesn't mean it won't impact a large number of people.

In fact, what do you think about making that change only for v10 + since 9.x and prior has been like this for a long time.

Reply:
Sure, that makes sense. I'll update or raise another PR with those changes.

With regards to the tooling scripts being released from Sitecore, that is great news. I did notice that the development.ps1 scripts in the Docker image are significantly different. I don't if they will play nicely with the Sitecore 9 images, which Sitecore has never officially provided, so scripts will possibly still be required.

Reply:
In that case is it still necessary to introduce the breaking change if they're not going to get pulled forward? ie: moved to another repo?

The developer's local docker-compose file will be drastically different between 9.x and 10.x anyway, so the move to standardize on c:\deploy vs c:\src might be less important now. The risk (breaking change) vs reward might not be worth it?

Reply:
I don't know what the changes will be so can't really comment. Happy for you to leave it open for a few days/weeks, and then reject the PR if the other changes. This PR was mostly to raise and resolve the issue that the difference between the Community Images and Sitecore Images were themselves introducing a breaking and unexpected change.